gripper_input_numbers: [1106, 1105]
gripper_positions: [0.000, 0.006]
